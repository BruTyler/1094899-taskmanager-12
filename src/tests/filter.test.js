import FilterView from '../view/filter.js';

it(`Filter rendering`, () => {
  const filterItems = [
    {title: `all`, count: 1},
    {title: `archive`, count: 2}
  ];

  const generatedTree = new FilterView(filterItems).getTemplate();

  expect(generatedTree).toMatchSnapshot();
});
