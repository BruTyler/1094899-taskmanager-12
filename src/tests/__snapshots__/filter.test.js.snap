// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Filter rendering 1`] = `
"<section class=\\"main__filter filter container\\">
    <input
      type=\\"radio\\"
      id=\\"filter__all\\"
      class=\\"filter__input visually-hidden\\"
      name=\\"filter\\"
      
      
      value=\\"ALL\\"
    />
    <label for=\\"filter__all\\" class=\\"filter__label\\">
      all <span class=\\"filter__all-count\\">1</span></label
    ><input
      type=\\"radio\\"
      id=\\"filter__archive\\"
      class=\\"filter__input visually-hidden\\"
      name=\\"filter\\"
      
      
      value=\\"ARCHIVE\\"
    />
    <label for=\\"filter__archive\\" class=\\"filter__label\\">
      archive <span class=\\"filter__archive-count\\">2</span></label
    >
  </section>"
`;
